landlab/components/sink_fill/tests/conftest.py

```diff
@@ -0,0 +1,162 @@
+import pytest
+import numpy as np
```

Comment:
~with a filename of conftest.py is this discoverable? Or is there somewhere I missed that indicated that this file should be tested?~

Reply:
never mind. I've now learned what conftest.py. I've left my comment and this link to info

https://docs.pytest.org/en/2.7.3/plugins.html?highlight=re

so that if others have the same questions  they might find this.